timesheets/Code.gs

```diff
@@ -0,0 +1,166 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+// GLOBAL VARS
+var HRS_START = 4;
+var HRS_END = 8;
+var HRLY_PAY_COLUMN = 9;
+var APPROVAL_COLUMN = 11;
+var EMAIL_COLUMN = 3;
+
+/**
+* Creates the menu item "Run Functions" so user can run scripts on drop-down.
+*/
+function onOpen(e) {
+    var ui = SpreadsheetApp.getUi();
+    ui.createMenu('Run Functions')
+    .addItem('Calculate Pay', 'calculatePay')
+    .addSeparator()
+    .addItem('Add Approval Column', 'addApprovalColumn')
+    .addSeparator()
+    .addItem('Send Emails', 'sendEmails')
+    .addToUi();
+}
+
+/**
+* Runs the script "Send Emails". Sends e mails to every employee notifying 
+* them whether or not their timesheet has been approved.
+* Checks + updates "NOTIFIED" status so employers do not send e mails more than once.
+*/
+function sendEmails() {
+   
+    var sheet = SpreadsheetApp.getActiveSheet();
+    var lastRow = sheet.getLastRow();
+    var lastCol = sheet.getLastColumn();
+    // lastCol here is the NOTIFIED column
+  
+  
+    // goes through the whole email column
+    for (i =2; i <= lastRow; i++) { 
+      
+      // don't notify twice
+      var notified_status = sheet.getRange(i, lastCol).getValue();
+      if (notified_status == "NOTIFIED") {
+        continue;
+      }
+       
+      
+      var range = sheet.getRange(i, 3);
+      var email = range.getValue();
+      var dropdown_value = sheet.getRange(i,APPROVAL_COLUMN);    
+      
+      // approval email
+      if (dropdown_value.getValue() == "APPROVED") {
+        var message = "Your timesheet has been approved";
+        var subject = 'TimeSheet Approval';
+      }
+      
+      else if (dropdown_value.getValue() == "NOT APPROVED") {
```

Comment:
I can't find it in the style guide (it might be there), but we usually always put the else on the same line as the previous curly brace, separated by a space.

```
if () {
  // ...
} else if () {
  // ...
} else {
  // ...
}
```